.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
```

Comment:
This disables the ability to create blank issues. With the changes in this PR only bug reports or feature requests will be opened. For any kind of other more loose discussions we're attaching here a link to the Discussions page.